packages/flutter/lib/src/material/carousel.dart

```diff
@@ -146,12 +199,37 @@ class Carousel extends StatefulWidget {
   /// Defaults to false.
   final bool reverse;
 
+  /// Whether we allow the "squished" item to expand to the max size.
+  ///
+  /// If this is false, the layout of the carousel doesn't change. This is especially
+  /// useful when the carousel has a centered-hero layout in which the max item
+  /// is in the middle and at least one small item on each side, the first and
+  /// the last item cannot expand to the max size. If this is true, there will
+  /// be some space before/after the first/last item coming so every items have
+  /// a chance to be fully expanded.
+  ///
+  /// Defaults to true on hero layout and false on multi-browse layout.
```

Comment:
Both the the unnamed constructor and the weighted constructor use true as the default.
So this statement is incorrect for `false on multi-browse layout`, since there is no "false" as default anywhere. Either we update this statement or we add a `multiBrowse` constructor that uses `false` as default.